domain/src/main/java/co/nimblehq/compose/crypto/domain/model/CoinItem.kt

```diff
@@ -1,18 +1,21 @@
 package co.nimblehq.compose.crypto.domain.model
 
+import java.math.BigDecimal
+
 data class CoinItem(
     val id: String,
     val symbol: String,
     val coinName: String,
-    val currentPrice: Int,
-    val marketCap: Long,
+    val image: String,
+    val currentPrice: BigDecimal,
+    val marketCap: BigDecimal,
     val marketCapRank: Int,
-    val totalVolume: Long,
-    val high24h: Long,
-    val low24h: Long,
-    val priceChange24h: Double,
+    val totalVolume: BigDecimal,
+    val high24h: BigDecimal,
+    val low24h: BigDecimal,
+    val priceChange24h: BigDecimal,
     val priceChangePercentage24h: Double,
```

Comment:
Regarding [this](https://github.com/nimblehq/jetpack-compose-crypto/pull/26#discussion_r963222963) Shall we change the type from `Double` to `BigDecimal,` same for other fields? I'm good. If you want to update on the next `PR` 🚀

Reply:
@Wadeewee I only recommend to use BigDecimal for pricing fields. This is percentage field so it's not the same thing, double is enough.